src/random_test.cpp

```diff
@@ -1176,11 +1176,12 @@ bool execute_sql(std::string sql, Thd1 *thd) {
     thd->success = true;
     MYSQL_RES *result;
     result = mysql_store_result(thd->conn);
+    auto affected_rows_count = mysql_affected_rows(thd->conn);
```

Comment:
move it inside the if block. so we avoid calling if log_all || log_success is not passed